### ISSUE TITLE
ValueTuples don't show up properly

### ISSUE THREAD

**Issue by benjamin-hodgson** [dotnet/docfx#2512]
**DocFX Version Used**: 2.32.2

**Template used**: `default`

**Steps to Reproduce**:

Generate API docs for a method which features a `ValueTuple` in its type. I happened to spot it with an implicit conversion:
```csharp
public static implicit operator Attr((string name, string value) nameValue) { /* ... */ }
```

**Expected Behavior**:

The correct tuple type shows up in the generated HTML

**Actual Behavior**:

The type is missing:

![](https://i.imgur.com/ISL8WOp.jpg)

**Comment by leandroltavares** [comment]
It also happens in return types

`public static IEnumerable<(string, Match)> GetFilesByRegex(string path, string pattern, SearchOption searchOption)`

is shown as 

`public static IEnumerable<> GetFilesByRegex(string path, string pattern, SearchOption searchOption)`

**Comment by werwolfby** [comment]
I’m working on this, and I have few promissing results: [#117](
https://github.com/nuke-build/nuke/issues/117)

**Comment by werwolfby** [comment]
There is some small issue in Roslyn, that do not add right spaces in some cases:
![image](https://user-images.githubusercontent.com/705754/44307907-a9eaa200-a3b3-11e8-97cd-85ac0c8f1f30.png)

Beside that everything looks ok.

**Comment by werwolfby** [comment]
@superyyrrzz I think we can close this issue, because the fix (#3228) was already merged. 
It wasn't auto closed because there is not reference in commit message
